spec/Interoperation.tex

```diff
@@ -308,47 +308,7 @@ \subsubsection{Referring to External Structs Through Pointers}
 \label{Referring_to_External_Structs_Through_Pointers}
 \index{interoperability!C Structs!external!pointers to}
 
-An external type which is a pointer to a \chpl{struct} can be referred to from
-Chapel using an external \chpl{class} declaration.  External class declarations
-have the following syntax.
-\begin{syntax}
-external-class-declaration-statement:
-  `extern' external-name[OPT] simple-class-declaration-statement
-\end{syntax}
-External class declarations are similar to external record declarations as
-discussed above, but place additional requirements on the C code.
-
-For example, given the declaration
-\begin{chapel}
-  extern class D {
-    var x: real;
-  }
-\end{chapel}
-\noindent
-the requirements on the corresponding C code are:
-\begin{enumerate}
-\item There must be a struct type that is typedef'd to have the name \chpl{\_D}.
-\item A pointer-to-\chpl{\_D} type must be typedef'd to have the name \chpl{D}.
-\item The \chpl{\_D} struct type must contain a field named \chpl{x} of type \chpl{double}.
-\end{enumerate}
-\noindent
-Like external records/structs, it may also contain other fields
-that will simply be ignored by the Chapel compiler.
-
-The following C typedef would fulfill the external Chapel class
-declaration shown above:
-\begin{chapel}
-   typedef struct __D {
-     double x;
-     int y;
-   } _D, *D;
-\end{chapel}
-where the Chapel compiler would not know about the 'y' field and
-therefore could not refer to it or manipulate it.
-
-If the optional \sntx{external-name} is supplied, then it is used verbatim as
-the exported class symbol.
-
+%INIT TODO: clearly state extern classes are not supported?
```

Comment:
Be sure to call out in your PR message that you also handle the removal of extern classes from the spec